### PR TITLE
Transient registration of sub operations

### LINKED ISSUE #18: Allow transient IoC registrations to be passed in
For the registrations of the sub operations to work correctly the `IocHandler` must accept a second action for transient registrations.

### PR DESCRIPTION
**Pull request by marco-bertschi**
Fixes #18

### CODE REVIEW

**Comment by pascalberger** [comment]
Please rebase on `release/1.0.0` branch and set target branch to `release/1.0.0` on this pull request.

**Comment by marco-bertschi** [comment]
> 
> 
> Please rebase on `release/1.0.0` branch and set target branch to `release/1.0.0` on this pull request.

Done.

File: src/BBT.StructureTools/Initialization/IoCHandler.cs [hunk 1]
```diff
@@ -41,102 +42,104 @@ public static IocHandler Instance
 
         /// <summary>
         /// Executes the binding between interfaces and implementations within BBT.StructureTools
-        /// using the <paramref name="registrationAction"/> for each pair.
+        /// using the <paramref name="singletonRegistrationAction"/> for each pair.
         /// </summary>
```
Comment (pascalberger) [inline src/BBT.StructureTools/Initialization/IoCHandler.cs:46]: Document parameters

**Review by pascalberger — APPROVED** [review]
Just one small thing. Otherwise LGTM

File: src/BBT.StructureTools/Initialization/IoCHandler.cs (hunk 1 above)
Comment (marco-bertschi) [reply src/BBT.StructureTools/Initialization/IoCHandler.cs:46]: Documented

**Pull request merged**